joblib/parallel.py

```diff
@@ -23,31 +21,20 @@
     import pickle
 
 from ._multiprocessing_helpers import mp
-if mp is not None:
-    from .pool import MemmapingPool
-    from multiprocessing.pool import ThreadPool
 
 from .format_stack import format_exc, format_outer_frames
 from .logger import Logger, short_format_time
 from .my_exceptions import TransportableException, _mk_exception
 from .disk import memstr_to_kbytes
+from ._parallel_backends import (MultiProcessingBackend, ThreadingBackend,
+                        SequentialBackend)
```

Comment:
This looks like a PEP8 to me. You should spend some time to configure your editor to run flake8 in order to highlight this kind of problems while you are editing.